tfx/dependencies.py

```diff
@@ -154,7 +154,8 @@ def make_extra_packages_test():
   # Note: It is okay to pin packages to exact versions in this list to minimize
   # conflicts.
   return make_extra_packages_airflow() + make_extra_packages_kfp() + [
-      'pytest>=5,<7',
+      'pytest>=5,<=8',
+      'pytest-subtests==0.13.1',
```

Comment:
What do we need `pytest-subtests` for?

Reply:
I believe there are `unittest` subtests, and they don't work with `pytest` unless this included